osu.Game.Rulesets.Taiko/Configuration/TaikoTouchControlScheme.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
we don't use nullable disable in new files

Reply:
Done